website/addons/dataverse/static/dataverseFangornConfig.js

```diff
@@ -254,25 +255,36 @@ function _fangornDataverseTitle(item, col) {
 
 function _fangornColumns(item) {
     var tb = this;
-    var selectClass = '';
     var columns = [];
     columns.push({
         data : 'name',
         folderIcons : true,
         filter : true,
-        css: selectClass,
         custom: _fangornDataverseTitle
     });
-
-    if (tb.options.placement === 'project-files') {
+    if(tb.options.placement === 'project-files') {
```

Comment:
This line and the other conditional below should both have a space between `if` and `(`, as per [cosdev](http://cosdev.readthedocs.io/en/latest/style_guides/javascript.html)